rules/apps.libsonnet

```diff
@@ -13,6 +13,16 @@
             // Reduces cardinality of this timeseries by #cores, which makes it
             // more useable in dashboards.  Also, allows us to do things like
             // quantile_over_time(...) which would otherwise not be possible.
+            record: 'node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate5m',
```

Comment:
Need to assess the impact of running both of these rules within the same evaluation group, let's test this internally before merging. It may be that each of these needs to be isolated to their own evaluation groups, but only if necessary to meet the evaluation deadline (usually 60 seconds).